### PR TITLE
Fix build failure introduced by #7595

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

Build is failing on feature branch

### How was it solved?

Fix the failing build

### How was it tested?

All Jenkins should pass

### CODE REVIEW

File: framework/src/modules/token/cc_commands/cc_forward.ts [hunk 1]
```diff
@@ -116,7 +115,7 @@ export class CCForwardCommand extends BaseCCCommand {
 
 		escrowData.amount -= params.amount + params.forwardedMessageFee;
 		await escrowStore.set(methodContext, escrowKey, escrowData);
-		const localTokenID = Buffer.concat([CHAIN_ID_ALIAS_NATIVE, localID]);
+		const localTokenID = Buffer.concat([Buffer.alloc(0), localID]);
```
Comment (Incede) [inline framework/src/modules/token/cc_commands/cc_forward.ts:118]: Under endpoint.ts `CHAIN_ID_ALIAS_NATIVE = Buffer.from([0, 0, 0, 1])` is different from the substitute used here. Which logic decides this ?

Comment (shuse2) [reply framework/src/modules/token/cc_commands/cc_forward.ts:118]: Concept of `ALIAS` is removed and also `cc_forward` is removed, so it's just temporal replacement until this files are removed / replaced in the new issues

File: framework/src/modules/token/cc_method.ts [hunk 2]
```diff
@@ -198,7 +192,7 @@ export class TokenInteroperableMethod extends BaseInteroperableMethod {
 		escrowData.amount -= totalAmount;
 		await escrowStore.set(ctx, escrowKey, escrowData);
 
-		const localTokenID = Buffer.concat([CHAIN_ID_ALIAS_NATIVE, localID]);
+		const localTokenID = Buffer.concat([Buffer.alloc(0), localID]);
```
[the remark this answers is not in the document]
Comment (shuse2) [reply framework/src/modules/token/cc_method.ts:195]: yes also the same, it is temporal value and the concept of ALIAS is removed so update LIP has different logic here

**Review by has5aan — APPROVED** [review]

File: framework/src/modules/token/types.ts [hunk 3]
```diff
@@ -69,6 +69,9 @@ export interface InteroperabilityMethod {
 	): Promise<boolean>;
 	error(methodContext: MethodContext, ccm: CCMsg, code: number): Promise<void>;
 	terminateChain(methodContext: MethodContext, chainID: Buffer): Promise<void>;
-	getChannel(methodContext: MethodContext, chainID: Buffer): Promise<{ messageFeeTokenID: Buffer }>;
+	getChannel(
+		methodContext: MethodContext,
+		chainID: Buffer,
+	): Promise<{ messageFeeTokenID: { chainID: Buffer; localID: Buffer } }>;
```
[the remark this answers is not in the document]
Comment (shuse2) [reply framework/src/modules/token/types.ts:75]: This is also temporal change. This is expected interface now in the original token module implementation, however, now we have `getMessageFeeToken` function as in the next line.
The new implementation should use this new function, but until all the code are updated, this interface is necessary to pass the build

**Review by Incede — APPROVED** [review]

**Pull request merged**